mscore/palettebox.cpp

```diff
@@ -63,6 +69,18 @@ PaletteBox::PaletteBox(QWidget* parent)
       hlSearch->setContentsMargins(5,0,5,0);
       hlSearch->addWidget(_searchBox);
 
+
+      QFile file(":/default.txt");
```

Comment:
The file `default.txt` will not be found at runtime unless you specifically ask for it to be bundled with the executable. To do this, simply add it to the list of files in [mscore/musescore.qrc](https://github.com/musescore/MuseScore/blob/55acd81bb68a50ff7dff2338eb22b1c40580d722/mscore/musescore.qrc) (a good place to put it is right after the [entry for revision.h](https://github.com/musescore/MuseScore/blob/55acd81bb68a50ff7dff2338eb22b1c40580d722/mscore/musescore.qrc#L40) as they are both in the same folder). See [Qt Resource System](https://doc.qt.io/qt-5/resources.html) to learn more.

Reply:
Thanks for the help I was able to fix the TreeView on the Palette. I will soon start the work on parsing basic.xml using the XML reader to create a model which will collect the palette names.

Reply:
![image](https://user-images.githubusercontent.com/41850468/59683222-5f190100-91f5-11e9-85a8-a87665df9f62.png)